python/lsst/ip/isr/overscan.py

```diff
@@ -221,32 +222,26 @@ def run(self, exposure, amp, isTransposed=False):
             maskIm = exposure.getMaskedImage()
             maskIm = maskIm.Factory(maskIm, parallelOverscanBBox)
 
-            # The serial overscan correction has removed the majority
-            # of the signal in the parallel overscan region, so the
-            # mean should be close to zero.  The noise in both should
-            # be similar, so we can use the noise from the serial
-            # overscan region to set the threshold for bleed
-            # detection.
-            thresholdLevel = self.config.numSigmaClip * serialResults.overscanSigmaResidual
-            makeThresholdMask(maskIm, threshold=thresholdLevel, growFootprints=0)
-            maskPix = countMaskedPixels(maskIm, self.config.maskPlanes)
-            xSize, ySize = parallelOverscanBBox.getDimensions()
-            if maskPix > xSize*ySize*self.config.parallelOverscanMaskThreshold:
-                self.log.warning('Fraction of masked pixels for parallel overscan calculation larger'
-                                 ' than %f of total pixels (i.e. %f masked pixels) on amp %s.',
-                                 self.config.parallelOverscanMaskThreshold, maskPix, amp.getName())
-                self.log.warning('Not doing parallel overscan correction.')
-            else:
-                parallelResults = self.correctOverscan(exposure, amp,
-                                                       imageBBox, parallelOverscanBBox,
-                                                       isTransposed=not isTransposed)
-
-                overscanMean = (overscanMean, parallelResults.overscanMean)
-                overscanMedian = (overscanMedian, parallelResults.overscanMedian)
-                overscanSigma = (overscanSigma, parallelResults.overscanSigma)
-                residualMean = (residualMean, parallelResults.overscanMeanResidual)
-                residualMedian = (residualMedian, parallelResults.overscanMedianResidual)
-                residualSigma = (residualSigma, parallelResults.overscanSigmaResidual)
+            # The serial overscan correction has removed some signal
+            # from the parallel overscan region, but that is largely a
+            # constant offset.  The collapseArray method now attempts
```

Comment:
It's calculated from the double-overscan corner, subtracting one value per row.  All rows should be the same within the read noise, as there's no real signal.  For the images I've been testing, the serial overscan median value is ~13000-15000, and subtracting it pulls the parallel overscan region down so that the parallel overscan median value is ~10-20.

The overscan code is far more confusing than I'd like (definitely my fault).  The parallel overscan uses the same code path as the serial, just with some extra transposes.  `correctOverscan` calls `fitOverscan`, which calls either `measureConstantOverscan` or `measureVectorOverscan`.  Continuing with the vector case, that method calls `collapseArray` for all except `MEDIAN_PER_ROW`, which instead calls `fitOverscanImage` (the C++ code), then the `fillMaskedPixels` method that is also called by `collapseArray`.  I'm not happy with the code-spaghetti I've made.